test/yaml_elixir_test.exs

```diff
@@ -144,14 +144,18 @@ defmodule YamlElixirTest do
       }
     """
 
-    assert_parse_string(json, %{
-      "a" => "a",
-      "b" => 1,
-      "c" => true,
-      "d" => nil,
-      "e" => "null",
-      "f" => 1.2
-    }, schema: :json)
+    assert_parse_string(
+      json,
+      %{
+        "a" => "a",
+        "b" => 1,
+        "c" => true,
+        "d" => nil,
+        "e" => "null",
+        "f" => 1.2
+      },
+      schema: :json
+    )
```

Comment:
Annotation: No logic change here, `mix format` made thee changes.  It also made some other changes.